plugins/modules/xbps.py

```diff
@@ -67,6 +67,21 @@
         type: bool
         default: true
         version_added: '0.2.0'
+    root:
+        description:
+            - The full path for the target root directory.
+        type: str
+        version_added: '10.1.0'
+    repository:
+        description:
+            - Repository URL(s) to prepend to the repository list for the
+              package installation.
+              The URL can be a URL to a repository for
+              remote repositories or a path for local repositories.
+        aliases: [repositories]
```

Comment:
Generally aliases should only be used if absolutely necessary. Here I would remove it, and potentially rename the option to `repositories`.